_targets.R

```diff
@@ -22,83 +23,83 @@ tar_option_set(packages = c("tidyverse",
-
-mapped_by_state_targets <- tar_map(
-  # Use state as suffix in branch target naming
-  names = state_abb,
-  # Use unlist = F so that we can reference only the branch targets from
-  unlist = FALSE,
-  # Task names passed into tar_map() via arg. 'values='
-  values = tibble(state_abb = states) %>%
-    mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_abb)),
-
-  # Target 1 - Subset to states in selected states list
-  tar_target(nwis_inventory, dplyr::filter(oldest_active_sites,
-                                           state_cd == state_abb)),
-  #Target 2 - Download data for given site
-  tar_target(nwis_data, get_site_data(nwis_inventory,
-                                      state_abb,
-                                      parameter)),
-  # Target 3 - Clean and summarize data for each state
-  tar_target(tally, tally_site_obs(nwis_data)),
-
-  # Target 4 - Produce timeseries plot for each state
-  tar_target(timeseries_png, plot_site_data(out_file = state_plot_files,
-                                            site_data = nwis_data,
-                                            parameter = parameter),
-             format = "file")
-)
+# states <- c("WI", "MN", "MI")
+
+states <- c('AL','AZ','AR','CA','CO','CT','DE','DC','FL','GA','ID','IL','IN','IA',
+            'KS','KY','LA','ME','MD','MA','MI','MN','MS','MO','MT','NE','NV','NH',
+            'NJ','NM','NY','NC','ND','OH','OK','OR','PA','RI','SC','SD','TN','TX',
+            'UT','VT','VA','WA','WV','WI','WY','AK','HI','GU','PR')
+
+parameter <- c('00060')
 
 ## List of Targets
+
 list(
-  # Identify oldest sites in NWIS data
+  # Fetch - Identify oldest sites in NWIS data
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
-  # Pull all targets in the tar_map() above
-  mapped_by_state_targets,
-
-  # Combine all state obs tallies (output of tar_map() )
-  tar_combine(obs_tallies, mapped_by_state_targets$tally,
-              command = combine_obs_tallies(!!!.x)),
+  # Fetch - Subset to states in selected states list
+  tar_target(nwis_inventory,
+             oldest_active_sites %>%
+               group_by(state_cd) %>%
+               tar_group(),
+             iteration = "group"),
+
+  # Fetch - Download data for given site
+  tar_target(nwis_data, retry::retry(get_site_data(nwis_inventory,
+                                                   nwis_inventory$state_cd,
+                                                   parameter), when = "Ugh, the internet data transfer failed!",
+                                     max_tries = 30),
+             pattern = map(nwis_inventory)
+             ),
+
+  # Process - Tally (clean and summarize) data for each state
+  tar_target(tally, tally_site_obs(nwis_data), pattern = map(nwis_data)),
+
+  # Vis - Produce timeseries plots for each state
+  tar_target(timeseries_png, plot_site_data(out_file = sprintf("3_visualize/out/timeseries_%s.png", unique(nwis_data$State)),
```

Comment:
This indentation is a little tricky to read. Can you fix it so that the `sprintf` is to the right of the `tar_target`?